ansible_collections/arista/avd/plugins/action/eos_designs_facts.py

```diff
@@ -195,7 +195,7 @@ def render_avd_switch_facts(self, avd_switch_facts_instances: dict) -> dict:
             try:
                 rendered_facts[host] = {"switch": avd_switch_facts_instances[host]["switch"].render()}
             except AristaAvdMissingVariableError as e:
-                msg = f"{e} is required but was not found for host '{host}'"
+                msg = f"{e}" if e.custom_message else f"{e} is required but was not found for host '{host}'"
```

Comment:
@ClausHolbechArista because of this - otherwise we get a crapy error message in facts that we don't get in structured_config:

```
 "Profile 'THIS-PROFILE-DOES-NOT-EXIST' applied under SVI 'TEST-SVI' does not exist in `svi_profiles`. is required but was not found for host 'failure-svi-parent-profile-does-not-exist'"
```